css/index.css

```diff
@@ -66,50 +66,49 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-  width: 880px;
+  width: 100%;
   /* TB LR */
   margin: 0 auto;
 }
 
-@media(max-width: 800px) {
+@media only screen and (max-width: 800px) {
```

Comment:
Nice use of units!
double check but I think this should be 
`@media screen only and ...`